apps/server/src/trpc/get-one.ts

```diff
@@ -95,13 +105,35 @@ export const getOne = createRouter()
     },
   })
 
+  /** get a user by email */
+  .query('user-by-email', {
```

Comment:
`/user/get-by-email` -> `/user-by-email`, to follow entity/{entityId}/action convention. I wouldn't mind having {email} for the middle part, but email doesn't work well with path params